website/src/routes/(layout)/[framework]/guides/kobalte.mdx

```diff
@@ -137,8 +137,8 @@ export function Select(props: SelectProps) {
       multiple={false}
       value={getValue()}
       onChange={setValue}
-      optionValue="value"
-      optionTextValue="label"
+      optionValue={(option: Option) => option.value}
```

Comment:
Thanks for the fix! Is the `: Option' declaration necessary? Isn't it inferred automatically?

Reply:
My project was not inferring it for some reason and gave me TS errors, but I think it should be inferred automatically in a normaly set up project. I might have broken my ts config aat some point and didnt realize it

Reply:
Hmm. I am not sure how to proceed. It would be nice to have a confirmation. If it is inferred, I recommend removing it.

Reply:
Ok ill check it out